packages/common/src/sites/_internal/capabilities/capabilityToFeatureMap.ts

```diff
@@ -0,0 +1,21 @@
+import { ICapabilityToFeatureMap } from "./types";
+
+/**
+ * Returns an array of ICapabilityToFeatureMap objects
+ * that define the projection of legacy capabilities
+ * to permission features.
+ *
+ * We use this map to manage foward and backard conversions
+ * and migration of legacy capablities to features when a
+ * site is fetched or updated
+ *
+ * NOTE: if you are porting a legacy capability over to
+ * workspaces, add it to this map
+ */
+export const capabilityToFeatureMap: ICapabilityToFeatureMap[] = [
+  {
+    capability: "hideFollow",
```

Comment:
maybe stab a comment in here explaining the current behavior - something like
```
// Currently if the hideFollow capability is present, the app interprets that to mean
// the following is disabled for the site, and thus no "Follow" buttons are displayed
// Permissions are structured in the positive, meaning that hub:site:feature:follow should
// return true if following is enabled, and false if it's disabled
```

Reply:
Since this where other devs will add more things, setting an example with detailed comments will help in the long run